crypto/cmp/cmp_status.c

```diff
@@ -67,10 +65,8 @@ const char *ossl_cmp_PKIStatus_to_string(int status)
  */
 OSSL_CMP_PKIFREETEXT *ossl_cmp_pkisi_get0_statusstring(const OSSL_CMP_PKISI *si)
 {
-    if (si == NULL) {
-        CMPerr(0, CMP_R_NULL_ARGUMENT);
-        return NULL;
-    }
+    if (!ossl_assert(si != NULL))
+        return 0;
```

Comment:
Fix introduced wrong error return value - should stay NULL here.

Reply:
Fixed